### DISCUSSION TITLE
Question about tables

### DISCUSSION THREAD

**Discussion by EduZeroum** [Budibase/budibase#1959] [Help]
Hello, i am doing my website and i want to have a "main" table that fills with my other tables. Is it possible? I have clients table , products table and i want to interconnect them. 

I really want this help.

**Comment by shogunpurple** [comment]
@EduZeroum could you give me an example?

Seems like relationships is what you are looking for: https://docs.budibase.com/data/data-types/relationships

**Reply by EduZeroum** [reply]
Is that exactly what i want , thank you so much i will explore a little bit. Thank you for the very fast answer ! :)

**Reply by EduZeroum** [reply]
Can i have you discord or any social media to interact more easier with you? :D

**Reply by shogunpurple** [reply]
https://github.com/Budibase/budibase/discussions/1719

Our discord server is not for support - mainly for product feedback and suggestions.

If you need help using budibase or are having issues, please keep using our github discussions forum 👍

**Reply by EduZeroum** [reply]
I think i have a bug when i put a relationship a weird bug... 
Link to video: https://streamable.com/lty46l

By the way the discord link server is expired.

**Reply by shogunpurple** [reply]
Whoops, here's another: https://discord.gg/dQwEjDdw

I think the problem there might be the fact you already have a column called "Produto" in the other table. Could you try with different column names in your relationship?

**Reply by EduZeroum** [reply]
Thank you for the discord link server, but i'm very confused about the relationships because how can i set the "Produto" to be related with the "main" table, i  tried to create a column named "Nome" (name) in the table "Produtos" (products) and created a main table with the clients name etc and "Produto" (product) but where i put the names to be a "list"... thats why i asked for your discord its more easier to explain my idea, my english is bad...

**Reply by shogunpurple** [reply]
Another streamable link would be helpful to show your setup. It doesn't appear from your screenshot that you have a "main" table, so I'm a little confused
